test/specs/commonTests/implementsShorthandProp.tsx

```diff
@@ -18,34 +18,47 @@ export default Component => {
     const { mapsValueToProp } = options
     const { displayName } = ShorthandComponent
 
+    /**
+     * @param {ReactWrapper} wrapper [mounted Component]
```

Comment:
lets use `-` as a separator - for the sake of being consistent with other places where JS doc is used

```js
/**
 * @param {ReactWrapper} wrapper - mounted Component
```